StaggeredCoupledProcesses/TH/ThermalDensityDrivenFlow2D/thermal_gravity_driven_flow2d.prj

```diff
@@ -0,0 +1,304 @@
+<?xml version="1.0" encoding="ISO-8859-1"?>
+<OpenGeoSysProject>
+    <mesh>mesh2D.vtu</mesh>
+    <geometry>gravity_driven.gml</geometry>
+
+    <processes>
+        <process>
+            <name>HeatConduction</name>
+            <type>HEAT_CONDUCTION</type>
+            <integration_order>2</integration_order>
+
+            <thermal_conductivity>lambda</thermal_conductivity>
+            <heat_capacity>c_p</heat_capacity>
+            <density>rho</density>
```

Comment:
It is a Parameter data name, and its value is given in the segment of <parameters>.